docs/docs/tutorials/set-model-parameters.md

```diff
@@ -3,5 +3,82 @@ sidebar_position: 4
 ---
 
 # 2. Set Model Parameters
+An example of specifying model parameters can be found [here](https://github.com/langchain4j/langchain4j-examples/blob/main/tutorials/src/main/java/_01_ModelParameters.java
+).
 
-Coming soon
\ No newline at end of file
+## What are Parameters in LLMs
+Depending on which model and which model provider you use, you can set a lot of parameters that will influence the model's output, speed, logging, etc.
+Typically, you will find all the parameters and their meaning on the provider's website.
+
+
+For example, OpenAI API's parameters can be found at https://platform.openai.com/docs/api-reference/chat
+and include options like
+
+| Parameter          | Description                                                                                                                                                                                                                                                                                              | Type      |
+|--------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| `modelName`        | The name of the model to use (gtp-3.5-turbo, gpt-4-1106-preview, ...)                                                                                                                                                                                                                                    | `String`  |

```

Comment:
I am not sure we should specify concrete parameters and descriptions here as they might be outdated quite soon and we dont want to maintain that

Reply:
I'm trying to have the dive-in threshold as low as possible, so personally I prefer that people have an easy direct way to know the parameters (on openAI website they don't exactly come in easy findable format), even if some parameter gets outdated from time to time. I'm happy to maintain it overall. I added a (most up-to-date version) note next to the url to openAI so people know they can find the latest version there in case of issues.

Reply:
Listing a few params as example is ok, but keeping them all here is cumbersome. Link to the [api reference](https://platform.openai.com/docs/api-reference/chat) should be enough.